src/templates/homepage.html

```diff
@@ -0,0 +1,25 @@
+<div class="app-preview"></div>
+
+<div class="app-list">
+  {% defer (url=api('apps'), pluck='objects', as='apps', key='slug') %}
+    <ul class="clearfix">
```

Comment:
Instead of a clearfix we might be able to use a non-floated list for the app tiles using flexbox. See my earlier comment about that.